askama_shared/src/lib.rs

```diff
@@ -45,17 +43,17 @@ impl<'a> Config<'a> {
             RawConfig::from_toml_str(s)?
         };
 
-        let (dirs, default_syntax) = match raw.general {
+        let (dirs, default_syntax, generated) = match &raw.general {
             Some(General {
                 dirs,
                 default_syntax,
+                generated,
             }) => (
-                dirs.map_or(default_dirs, |v| {
-                    v.into_iter().map(|dir| root.join(dir)).collect()
-                }),
+                dirs.as_deref().unwrap_or(DEFAULT_DIRS),
                 default_syntax.unwrap_or(DEFAULT_SYNTAX_NAME),
+                generated.unwrap_or(false),
```

Comment:
```suggestion
                generated,
```

(if the `Option` is removed)